### PR TITLE
add upper bound on pyflakes to python-lang.-server

### PR DESCRIPTION
**Pull request by jjhelmus**
Add an upper bound on pyflakes <2.2.2 to the python-language-server
packages below version 0.31.10.

### CODE REVIEW

**Comment by jjhelmus** [comment]
This is preferred over moving these packages to the `broken` label.

File: recipe/gen_patch_json.py [hunk 1]
```diff
@@ -478,6 +479,17 @@ def _gen_new_index(repodata, subdir):
...
                 record['depends'][i] = 'msgpack-python <1.0.0'
 
+        # python-language-server <=0.31.9 requires pyflakes <2.2.2
+        # included explicitly in 0.31.10+
+        # https://github.com/conda-forge/python-language-server-feedstock/pull/50
+        version = record['version']
+        pversion = pkg_resources.parse_version(version)
+        v0_31_9 = pkg_resources.parse_version('0.31.9')
+        if record_name == 'python-language-server' and pversion <= v0_31_9:
+            if 'pyflakes >=1.6.0' in record['depends']:
```
Comment (isuruf) [inline recipe/gen_patch_json.py:489]:
```suggestion
        if record_name == 'python-language-server':
            pversion = pkg_resources.parse_version(version)
            v0_31_9 = pkg_resources.parse_version('0.31.9')
            if pversion <= v0_31_9 and 'pyflakes >=1.6.0' in record['depends']:
```

**Comment by ccordoba12** [comment]
@conda-forge/core, could you merge this one?

**Comment by ccordoba12** [comment]
Thanks a lot @isuruf!

**Pull request merged**